doc/arch/adr-001-feature-flags.md

```diff
@@ -0,0 +1,57 @@
+# 001 - Feature Flags
+
+## Context
+
+We need to add a mechanism to toggle features in MMW so that we can deploy
+multiple implementations from the same codebase. For example, in BiG CZ,
+we'll need to disable Analyze mode and other MMW-specific features such as
+ITSI integration.
+
+Primary deciding factors:
+
+* How will this impact development?
+* How easy will it be to switch from MMW to BiGCZ?
+* How will this impact deployments?
```

Comment:
Is the idea that all development for MMW or BiG CZ will occur within this same repository, i.e., no repository forking?